docs/security-best-practices.md

```diff
@@ -1,324 +1,82 @@
-considerations for the Collector. This document is intended for both end-users
-and component developers. It assumes at least a basic understanding of the
-Collector architecture and functionality.
+considerations for the Collector. This document is intended for component
+developers. It assumes at least a basic understanding of the Collector
+architecture and functionality.
 
-> Note: Please review the [configuration
-> documentation](https://opentelemetry.io/docs/collector/configuration/)
+> Note: Please review the
+> [configuration documentation](https://opentelemetry.io/docs/collector/configuration/)
 > prior to this security document.
 
-## TL;DR
-
-### End-users
-
-- Configuration
-  - SHOULD only enable the minimum required components
-  - SHOULD ensure sensitive configuration information is stored securely
-- Permissions
-  - SHOULD not run Collector as root/admin user
-  - MAY require privileged access for some components
-- Receivers/Exporters
-  - SHOULD use encryption and authentication
-  - SHOULD limit exposure of servers to authorized users
-  - MAY pose a security risk if configuration parameters are modified improperly
-- Processors
-  - SHOULD configure obfuscation/scrubbing of sensitive metadata
-  - SHOULD configure recommended processors
-- Extensions
-  - SHOULD NOT expose sensitive health or telemetry data
+Security documentation for end users can be found on the OpenTelemetry
+documentation website:
 
-> For more information about securing the OpenTelemetry Collector, see
-> [this](https://medium.com/opentelemetry/securing-your-opentelemetry-collector-1a4f9fa5bd6f)
-> blog post.
+- [Collector configuration best practices](https://opentelemetry.io/docs/security/config-best-practices/)
+- [Collector hosting best practices](https://opentelemetry.io/docs/security/hosting-best-practices/)
 
-### Component Developers
+## TL;DR
 
 - Configuration
   - MUST come from the central configuration file
   - SHOULD use configuration helpers
 - Permissions
   - SHOULD minimize privileged access
-  - MUST document what required privileged access and why
+  - MUST document what requires privileged access and why
 - Receivers/Exporters
   - MUST default to encrypted connections
   - SHOULD leverage helper functions
 - Extensions
   - SHOULD NOT expose sensitive health or telemetry data by default
 
+> For more information about securing the OpenTelemetry Collector, see
+> [this blog post](https://medium.com/opentelemetry/securing-your-opentelemetry-collector-1a4f9fa5bd6f).
+
 ## Configuration
 
 The Collector binary does not contain an embedded or default configuration and
 MUST NOT start without a configuration file being specified. The configuration
-file passed to the Collector MUST be validated prior to be loaded. If an
+file passed to the Collector MUST be validated prior to being loaded. If an
 invalid configuration is detected, the Collector MUST fail to start as a
 protective mechanism.
```

Comment:
@mx-psi Hi and Happy New Year! 

I think this paragraph is pretty outdated, but I'm not sure how much of it is still accurate. I think based on #886 and #910 that none of it is true, but I could use your expertise. Thanks!

Reply:
The paragraph is still correct.

Reply:
Okay, great. Thanks for confirming!